resources/views/components/slideover/partials/header.blade.php

```diff
@@ -0,0 +1,21 @@
+<div class="slideover-header bg-primary py-5">
+    <div class="px-5">
+        <div class="relative flex items-center justify-center">
+            @if ($hasParent)
+                <label class="absolute left-0 top-1/2 -translate-y-1/2 cursor-pointer text-white" for="{{ $id }}">
```

Comment:
When this label is clicked the 
`<input @checked($open) id="{{ $id }}" v-on:change="toggleScroll($event.target.checked)" class="peer hidden" type="checkbox"> `
Will change and it will remove the scrolllock on the body so you are be able to scroll again.